integration/docker/entrypoint.sh

```diff
@@ -159,7 +159,16 @@ function setup_for_dynamic_non_root {
       mkdir -p /journal
       chown -R ${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} /opt/* /journal
       chmod -R g=u /opt/* /journal
-      exec su ${ALLUXIO_USERNAME} -c "/entrypoint.sh ${ARGS}"
+      # Chmod the dirs of tiered stores for alluxio worker
+      # to ensure write permission for non-root user.
+      if [[ "$1" == "worker" || "$1" == "worker-only" ]]; then
+        echo "$ALLUXIO_JAVA_OPTS" | tr ' ' '\n' | \
+          grep "alluxio.worker.tieredstore.level[0-9].dirs.path" | \
+          cut -d '=' -f 2 | \
+          grep -Ev "^$" | \
```

Comment:
this value may also contain a comma-delimited list of directories. We should handle that case too.

Reply:
@ZacBlanco, hi, I updated this script to support comma-delimited list of directories. thanks for your advice!